Presentation/Resource/ContentType.php

```diff
@@ -114,35 +115,39 @@ public static function getResourceClassFromType(string $contentType): string
         return static::CLASSES[$contentType] ?? static::CLASSES['_'];
     }
 
-    public function getResource(string $path, string $relType = '', bool $storeInCache = true): GenericResource
+    public function getResource(string $path, string $relType = '', $external = false, bool $storeInCache = true): ResourceInterface
     {
-        $path = static::resolveAbsolutePath($path);
+        $path = !$external ? static::resolveAbsolutePath($path) : $path;
```

Comment:
```suggestion
        $path = $external ? $path : static::resolveAbsolutePath($path);
```

May be easier to read